src/test/java/bio/terra/ChildContentHandlerTest.java

```diff
@@ -14,7 +14,7 @@ public void acceptEventTest() {
 
     try {
       StorageObjectData storageObjectData =
-          GsonWrapper.getInstance().fromJson(MOCK_EVENT_GZIP, StorageObjectData.class);
+          GsonWrapper.convertFromClass(MOCK_EVENT_GZIP, StorageObjectData.class);
       ChildContentHandler handler = new ChildContentHandler(storageObjectData);
```

Comment:
please replace all `System.out.println` in this file with a logger

Reply:
Replaced all `System.out.println` by logger.